checks.d/couchbase.py

```diff
@@ -172,7 +172,12 @@ def get_data(self, server, instance):
         url = '%s%s' % (server, COUCHBASE_STATS_PATH)
 
         # Fetch initial stats and capture a service check based on response.
-        service_check_tags = ['instance:%s' % server]
+        service_check_tags = instance.get('tags', [])
+        if service_check_tags is None:
+            service_check_tags = []
+        else:
+            service_check_tags = list(set(service_check_tags))
```

Comment:
@hkaj I made the changes in this way to keep service tag collection consistent with how tags are collected elsewhere in this check. See:

 https://github.com/arzarif/dd-agent/blob/b5932ebabf73765e25c2a330ae41a9bf932fc789/checks.d/couchbase.py#L152-L159